teps/README.md

```diff
@@ -0,0 +1,9 @@
+# Tekton Enhancement Proposals (TEPs)
+
+A Tekton Enhancement Proposal (TEP) is a way to propose, communicate
+and coordinate on new efforts for the Tekton project.  You can read
+the full details of the project in
+[TEP-1](0001-tekton-enhancement-proposal-process.md).
```

Comment:
once we agree on TEP-1 we'd probably duplicate some of those docs in the main readme maybe?

Reply:
Ah, we might write a better introduction / guide of TEPs in the main readme yeah :wink: